src/Symfony/Component/Mime/Tests/RawMessageTest.php

```diff
@@ -46,6 +46,21 @@ public function testSerialization(mixed $messageParameter, bool $supportReuse)
         }
     }
 
+    /**
+     * @dataProvider provideMessages
+     */
+    public function testToIterable(mixed $messageParameter, bool $supportReuse)
+    {
+        $message = new RawMessage($messageParameter);
+        $this->assertEquals('some string', implode('', iterator_to_array($message->toIterable())));
+
+        // in 7.0, the test with a generator will throw an exception
```

Comment:
you should be asserting the deprecation for cases not supporting reuse. This would prove you that it currently does not get triggered.